Azure/sonic-buildimage
# [Arista] Add DPB for 7060CX-32S

#### Why I did it
- To start support of dynamic port breakout as the norm for Arista platforms.
- Add a DPB hwsku for the 7060CX-32S

#### How I did it
- Expand platform.json for the 7060CX-32S
- Added a new hwsku specifically for DPB
- Added a flex Broadcom configuration

#### How to verify it
- Switch boots up with the configuration with ports up in normal 32x100G
- Switch is also able to boot up if hwsku.json is changed to default to other breakout modes, and I see 4x10G interfaces up
- Currently only initial configuration with hwsku.json works. Breakout via CLI is blocked on Azure/sonic-buildimage#7311
- Port tuning for this hwsku and the flex Broadcom config will be done at a later time, likely via media settings

#### Which release branch to backport (provide reason below if selected)




#### Description for the changelog



#### A picture of a cute animal (not mandatory but encouraged)
